### PR TITLE
README Documentation update

### PR DESCRIPTION
**Pull request by dondreojordan**

### CODE REVIEW

**Review by sberniz — COMMENTED** [review]
Looks good

File: README.md [hunk 1]
```diff
@@ -43,71 +68,24 @@ We're pulling data from similiar APIs(All locations V2 - https://raw.githubuserc
...
-# Environment Variables
+### Environment Variables
 
 In order for the app to function correctly, the user must set up their own environment variables. There should be a .env file containing the following:
 
-    *  PRAW_CLIENT_ID  - keys for Reddit API
-    *  PRAW_CLIENT_SECRET - keys for Reddit API
-    *  PRAW_USER_AGENT - keys for Reddit API
+    *  DBURLS  - Postgres database credentials
+    *  API URL - API url responsible for updating the database
```
Comment (jlong5795) [inline README.md:76]: Should this env have a space in it?

File: README.md [hunk 2]
```diff
@@ -145,13 +123,59 @@ We used Docker for ease of use when dealing with environmental dependancies
...
+
+Get and clean data from source(s). Insert processed data into the database. Data Science API recieve GET request from back end API. Send the data to the back end API for new information to be added to web database. The front end requests data from the back end and that data is serve to the map component and the dashboard. If use requests to sign-up/login, user is sent to the OKTA Identity Provider for authentification.   
+
+
+# Updates To Repo in Labs 28
+* Using a database instead of a .csv file
+* Added classification features based on associated tag words
 
-## Issue/Bug Request
+# End Points
```
Comment (jlong5795) [inline README.md:171]: I would just link to the live Swagger docs.

File: README.md [hunk 3]
```diff
@@ -1,16 +1,29 @@
...
+# Contributors
+
+| [Jason Long](https://github.com/jlong5795) | [Dondré Jordan](https://github.com/dondreojordan/) | [Santiago Berniz](https://github.com/sberniz/) | [Mariapaula Trujillo](https://github.com/mtruj013) |
+| :---: | :---: | :---: | :---: | 
+| [<img src="https://avatars3.githubusercontent.com/u/57104675?s=460&u=e7620554469deb924ac378097155cd45590cbdf9&v=4" width = "200" />](https://github.com/jlong5795) | [<img src="https://avatars2.githubusercontent.com/u/66181909?s=460&u=6e8dd07a4ee2d81ea91ff7f7bf92397e25348b43&v=4" width = "200" />](https://github.com/dondreojordan/) | [<img src="https://avatars3.githubusercontent.com/u/6207914?s=460&u=8bfaa068f7942170423371ff10e8f04f09f41e81&v=4" width = "200" />](https://github.com/sberniz/) | [<img src="https://avatars1.githubusercontent.com/u/17607612?s=460&u=1ade9305977ae41910de06971d36c7ebc339ef48&v=4" width = "200" />](https://github.com/mtruj013) |
+| Team Project Lead | Data Scientist | Data Scientist | Web Developer | Web Developer |
```
Comment (jlong5795) [inline README.md:15]: Technical Project Lead -> TPL

**Review by jlong5795 — CHANGES_REQUESTED** [review]
One question, one correction, and one suggestion.

File: README.md (hunk 3 above)
Comment (dondreojordan) [reply README.md]: Resolved.

File: README.md (hunk 1 above)
Comment (dondreojordan) [reply README.md:76]: The preview doesn't show this space.

**Review by dondreojordan — COMMENTED** [review]
Changes have been made. Review ready.

**Review by jlong5795 — APPROVED** [review]
Looks good. Thanks for the updates.

**Pull request merged**